governance/governance.md

```diff
@@ -0,0 +1,267 @@
+formal boundaries the same way the software does.
+
+In InnerSource there's also a global community[^5]. (In fact, it is
+an open source one, but that's another tale to be told on other
+occasions).
+
+But in contrast to open source, while the global InnerSource
+community shares information and resources, they do **not** share
+**their InnerSource production**. The presence of boundaries for
+the InnerSource production becomes thus very present and remarks
+the separation between the local community and the global one.
+
+### Activities
+
+As part of the open source spirit in which InnerSource roots,
+organizations are encouraged to engage with special interest groups
+(SIG) beyond their borders, and in particular with the global
+InnerSource community.
+
+Publicly asserting one's InnerSource practice and showcasing related
+experiences may make sense or not that much depending on the context
+of the corporation. Goals that benefit from such external engagement
+include:
+
+ - Plans to go open source in the future,
+ - InnerSource as hiring motivator
+
+Engagement with the internal community is a fundamental part of
+InnerSource. But as users of software of other teams, it is in
+self-interest to secure long-term support or knowledge transfer
+from upstream. The _code-consumers_ pattern[^6] is related to this.
+
+Usually, there's scant or no need for internal InnerSource
+purchase policies[^7], but if needed, it is a natural task for the
+InnerSource Program Office  (ISPO).
+
+## Make InnerSource Strategic for the organization
+
+Most times, InnerSource is a corporate cultural transformation. Such
+changes need time to overcome previous inertia and even longer to
+stabilize and become mainstream.
+
+Some bottom-up InnerSource initiatives die or languish after an
+initial unsuccessful push or because other priorities drain the
+attention before the cultural shift stabilizes.
+
+Making InnerSource strategic for the organization is the way to
+ensure its provided value is perceived as such.
+
+### Activities
+
+Ensure you design your InnerSource initiative and its implementation
+to feature important long-term corporate goals like innovation,
+digital sovereignty, and digital transformation. Think of your
+organization and its context and find other goals InnerSource may
+contribute towards. Then communicate it and get as much air cover
+from your executives as you can.
+
+
+[^1]: http://oss-watch.ac.uk/resources/governancemodels
+
+[^2]: https://ospo-alliance.org/ggi/
+
+[^3]: https://github.com/InnerSourceCommons/InnerSourcePatterns
+
+[^4]: https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/introducing-metrics-in-innersource.md
+
+[^5]: https://innersourcecommons.org/
+
+[^6]: https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/code-consumers.md
+
+[^7]: https://ospo-alliance.org/ggi/activities/open_source_procurement_policy/
```

Comment:
That's because you read it on a device. But this was meant to be a book with copies printed on paper, where hyperlinks don't work.

I've seen other pages (like the `introduction/framework` or the `infrastructure` chapter) providing clickable inline links, but most of the `introduction` chapter seems to go for footnotes, so I don't know.

Reply:
> ... this was meant to be a book with copies printed on paper, where hyperlinks don't work.

That is interesting. Is this information captured in writing somewhere, or information you got through the bitergia connection? ❤️ 

Also @rrrutledge what's your take on this?

Assuming that our current goal here is to publish an online book (and not a printed book), my preference would be to go for regular hyperlinks, rather than footnotes.

However for the moment it does not matter much either way. We can also leave them as footnotes for now and decide later what we want to do with them.